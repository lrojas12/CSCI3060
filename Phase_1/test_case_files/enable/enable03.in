login
admin
enable
Bob
00006
logout