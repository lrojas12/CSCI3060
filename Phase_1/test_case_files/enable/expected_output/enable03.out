Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information
Mode in which you wish to login as:
You are currently logged in as an administrator
Enter Account holder’s name:
You are currently logged in as acc_holder
Bank Account Number: acc_num
Balance: balance
Transaction PaymentPlan: plan
Status: status
ERROR: You need to be an administrator to enable an account.
You have successfully logged out of the administrator account